fluent-langneg/src/headers.js

```diff
@@ -0,0 +1,8 @@
+
+export default function parseAcceptedLanguagesHeaderIntoArray(string = '') {
+  if (typeof string !== 'string') throw new TypeError('Argument must be a string');
+  const tokens = string.split(',').map(t => t.trim());
+  return tokens.filter(t => t !== '').map(t => {
+    return t.split(';')[0];
```

Comment:
This doesn't take into effect the q-value nor the specificity of the locale tag. Are we fine with this?